apps/angular-example/src/app/app.component.ts

```diff
@@ -1,27 +1,22 @@
-import { Component, OnInit } from '@angular/core';
-import { Abby } from './abby';
+import { Component } from "@angular/core";
+import { FormControl } from "@angular/forms";
+import { shareReplay } from "rxjs";
+import { Abby } from "./abby";
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css'],
+  selector: "app-root",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.css"],
 })
-export class AppComponent implements OnInit {
-  angularTest: string;
-  angularFlag: boolean;
-  constructor(public readonly abby: Abby) {}
+export class AppComponent {
+  angularTest$ = this.abby.getVariant("AngularTest").pipe(shareReplay(1));
```

Comment:
consistency.. why shareReplay? why not everywhere? if everywhere, why doesnt the lib do it? no one wants to write this over and over again :shrug:

Reply:
abby is the abby-service created by the angular-lib? isnt it? so its under your full control ;) as mentioned above somewhere, this is a demo. this will be used by users. add comments everywhere or make it self-explaining. :)

Reply:
Oh wait nvm, you are right. The instance of the observable would be the same in this case, so we can use `shareReplay` here, even if it comes from the library